src/OpenTelemetry.AutoInstrumentation.Native/cor_profiler.cpp

```diff
@@ -60,6 +61,24 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
         }
     }
 
+#if defined(ARM64) || defined(ARM)
```

Comment:
Good to see that.
Not directly related to this PR.
Did you have chance to compile and execute instrumentation on ARM?

(I ma not asking to remove even if it is not fully working as expected or if you do not tried it yet).

Reply:
I haven't tried on Linux ARM64 yet, but I just tried it on OSX ARM64 and it _does not_ work. There are upstream changes made at a later point in time that should add support.